src/pages/Introduce/Introduce.scss

```
@@ -0,0 +1,69 @@
+@import '../../style/mixin';
+@import '../../style/variables';
+
+* {
+  box-sizing: border-box;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.introduce {
+  @include flexCenter;
+
+  .card {
+    display: flex;
+    flex-direction: column;
+    width: 140px;
+    height: 230px;
+    padding: 10px;
+    border: 1px solid $primary-black;
+    border-radius: 5px;
+
+    .logo {
+      width: 120px;
+      height: 120px;
+      margin: 0 0 10px;
+      border-radius: 5px;
+    }
+
+    & > p {
+      width: 120px;
+      text-align: left;
+    }
+
+    .my-name {
+      @include textStyle(12px, 700);
+      text-align: left;
+
+      &:hover {
+        color: $primary-green;
+      }
+    }
+
+    .description {
+      @include textStyle(11px, 400);
+      margin: 0;
+    }
+
+    .inputContainer {
+      @include flexCenter;
+      width: 120px;
+      margin-top: 5px;
+
+      .inputBox {
+        width: 90px;
+        height: 18px;
+      }
+
+      .inputBtn {
+        height: 18px;
+        padding: 0;
+        @include textStyle(10px, 400);
```

```suggestion
        @include textStyle(10px, 400);
        height: 18px;
        padding: 0;
```
include를 사용할 때는 스타일 블록의 가장 최상단에 사용해주는 것이 컨벤션입니다!